<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondForm</class>
 <widget class="QWidget" name="SecondForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form 2 - Transactions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Транзакции клиента</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLabel" name="labelDate">
       <property name="text">
        <string>Дата (гггг-мм-дд):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterDateEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="labelAmount">
       <property name="text">
        <string>Сумма (больше):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterAmountEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="labelType">
       <property name="text">
        <string>Тип:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterTypeCombo">
       <item>
        <property name="text">
         <string>Все</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>addition</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>opening</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>closing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>early closing</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFilter">
       <property name="text">
        <string>Фильтровать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="transactionTable"/>
   </item>
   <item>
    <layout class="QFormLayout" name="transactionFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelDepositId">
       <property name="text">
        <string>ID Вклада:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="depositIdEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAmountEdit">
       <property name="text">
        <string>Сумма:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="amountEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelDateEdit">
       <property name="text">
        <string>Дата (гггг-мм-дд чч:мм:сс):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dateEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelTypeEdit">
       <property name="text">
        <string>Тип:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="typeCombo">
       <item>
        <property name="text">
         <string>addition</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>opening</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>closing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>early closing</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="transactionButtonsLayout">
     <item>
      <widget class="QPushButton" name="btnAddTransaction">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEditTransaction">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDeleteTransaction">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnBack">
     <property name="text">
      <string>Back to Form 1</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>